scripts/src/bus.lua

```diff
@@ -2577,6 +2577,21 @@ if (BUSES["SS50"]~=null) then
 end
 
 
+---------------------------------------------------
+--
+--@src/devices/bus/technics/hdae5000.h,BUSES["TECHNICS"] = true
+---------------------------------------------------
+
+if (BUSES["TECHNICS"]~=null) then
+	files {
+		MAME_DIR .. "src/devices/bus/technics/kn5000_extension.cpp",
+		MAME_DIR .. "src/devices/bus/technics/kn5000_extension.h",
+		MAME_DIR .. "src/devices/bus/technics/hdae5000.cpp",
+		MAME_DIR .. "src/devices/bus/technics/hdae5000.h",
+	}
+end
```

Comment:
You want to trigger this on inclusion of the slot header, not one specific card.

Reply:
You mean it should be like this, instead?

`--@src/devices/bus/technics/kn5000_extension.h,BUSES["TECHNICS"] = true`

Reply:
Yes.  Look at any of the other optional buses.

Think about this logically:
* You need to trigger it for anything that may instantiate the slot, as that will require it whether any cards are used or not.
* The card headers `#include` the slot header anyway, so something trying to use a card without the slot would still trigger it.
* The slot header does not `#include` the card headers, so triggering on a card header won’t work if that particular card is not used.